<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAgentView</class>
 <widget class="QDialog" name="EditAgentView">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">EditAgentView { background: qlineargradient(spread:pad, x1:0.573909, y1:0.693, x2:0, y2:0, stop:0 rgba(176, 227, 100, 255), stop:1 rgba(255, 255, 255, 255)); }</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="confirmError">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lastNameLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastNameInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okayButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="firstNameLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailError">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstNameError">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstNameInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="userError">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordError">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="confirmLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastNameError">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
